faker/providers/date_time/bn_BD/__init__.py

```diff
@@ -0,0 +1,37 @@
+from .. import Provider as DateTimeProvider
+
+
+class Provider(DateTimeProvider):    
```

Comment:
There's some extra space at the end of this line that's making the linter fail. Could you remove it?
```suggestion
class Provider(DateTimeProvider):
```

Reply:
Thanks for the feedback. I have updated my pull-request.